src/main/java/uk/gov/ons/census/casesvc/model/dto/EventTypeDTO.java

```diff
@@ -12,6 +12,7 @@
   SAMPLE_LOADED,
   RM_UAC_CREATED,
   ADDRESS_NOT_VALID,
+  ADDRESS_MODIFIED,
```

Comment:
Remember we will also be receiving ADDRESS_TYPE_CHANGED on the same topic, so it's worth including in this PR

Reply:
still missing